src/client/style/oi.css

```diff
@@ -1,5 +1,162 @@
 /* OI Styles */
 
+.oi-error {
+  color:#dc3545;
+  font-size: 80%;
+  margin-top: 15px;
+}
+
+
+.oi-add-button {
+float: right;
+margin-right: 10px;
+margin-top: -7px;
+}
+.oi-modal-header {
+  background-color: #323e48;
+  color: #fff;
+  height: 32px;
+  padding-top: 4px;
+  }
+
+  .oi-modal-footer {
+    height: 90px;
+    }
+
+.oi-new-name {
+  right: 240px;
+  bottom: 10px;
+  }
+.connector-container {
+  height: 120px;
+  background-color: rgba(215, 218, 223, 0.36);
+  display: flex;
+  flex-direction: row;
+}
+
+.connector {
+  background-color:  #dee2e6;
+  text-align: center;
+  height: 70px;
+  margin-top: 25px;
+  margin-left: 75px;
+  margin-right: 65px;
+  width: 50px;
+  border:none;
+  border-radius:5px;
+}
+
+.connectorFocus {
+  background-color: #f5bb51;
+  color: #fff;
+  text-align: center;
+  height: 70px;
+  margin-top: 25px;
+  margin-left: 75px;
+  margin-right: 65px;
+  width: 105px;
+  border:none;
+  border-radius:5px;
+}
+  
+.connector2 {
+  background-color: #dee2e6;
+  text-align: center;
+  height: 70px;
+  margin-top: 25px;
+  margin-left: 120px;
+  margin-right: 125px;
+  width: 105px;
+  border:none;
+  border-radius:5px;
+}
+.connectorFocus2 {
+  background-color: #f5bb51;
+  color: #fff;
+  text-align: center;
+  height: 70px;
+  margin-top: 25px;
+  margin-left: 120px;
+  margin-right: 125px;
+  width: 105px;
+  border:none;
+  border-radius:5px;
+}
+.connector3 {
+  background-color: #dee2e6;
+  text-align: center;
+  height: 70px;
+  margin: 25px;
+  width: 150px;
+  border:none;
+  border-radius:5px;
+}
+.connector4 {
+  background-color: #f5bb51;
+  color: #fff;
+  text-align: center;
+  height: 70px;
+  margin: 25px;
+  width: 150px;
+  border:none;
+  border-radius:5px;
+}
```

Comment:
This seems wrong : you should find a way to condense all these classes into one. You can also have a general connector class and use other class that will overwrite or add some css features by using both classes in the frontend